run.go

```diff
@@ -307,6 +319,35 @@ func runAction(clicontext *cli.Context) error {
 		}
 		opts = append(opts, oci.WithTTY)
 	}
+    capAdd := clicontext.StringSlice("cap-add")
+    capDrop := clicontext.StringSlice("cap-drop")
+
+	if len(capAdd) > 0 || len(capDrop) > 0 {
+		if util.InStringSlice(capAdd, "ALL") {
+    		opts = append(opts, oci.WithAllCapabilities)
+    	}
+
+        if util.InStringSlice(capDrop, "ALL") {
+            opts = append(opts, oci.WithCapabilities(nil))
+        }
+        var caps []string
+        for _, c := range capAdd {
+            if strings.ToUpper(c) == "ALL" {
+                continue
+            }
+            caps = append(caps, "CAP_"+strings.ToUpper(c))
+        }
+    	opts = append(opts, oci.WithCapabilities(caps))
+
+    	caps = []string{}
```

Comment:
Please avoid reusing caps variable here. This should be a new variable like capsDrop